.github/tests-visual/visual-record.js

```diff
@@ -281,4 +291,7 @@ async function main() {
     await browser.close();
 }
 
-main();
+main().catch(error => {
+    console.error("::error ::" +error);
```

Comment:
wenn wir ein eigenes error object verwenden könnte und darin den dateinamen des screenshots im git repo mit gibt kann man hier in der fehlermeldung sogar die fehlermeldung zur datei zuordnen.

format wäre `"::error file=". $file ."::". $message`

dann taucht die fehlermeldung auch im Files-Changed tab auf

Reply:
Kann man wirklich bei Bildern auch Annotations setzen?

Reply:
Ich hab es nicht getestet. Aber da man errors auf file ebene abgeben kann, ohne zeile, bin ich davon ausgegangen.

Aber lässt sich einfach testen

Reply:
scheint nicht zu gehen lt. https://github.com/redaxo/redaxo/pull/4861/commits/999d34e7ef679a93c8dd08f9860fee9bc8cb6def